tests/h/h_api/bulk_api/command_batch_test.py

```diff
@@ -83,12 +82,12 @@ def add_commands(batch, commands):
                 pass
 
     @pytest.fixture
-    def command(self, user_attributes):
-        return CommandBuilder.user.upsert("acct:user@example.com", user_attributes)
+    def command(self, user_command):
+        return user_command
```

Comment:
Re-use the global fixtures, although it's still nice to point out here we don't care what this command is